text/0085-api-keys.md

```diff
@@ -201,8 +201,8 @@ Gives the total number of API keys that can be browsed.
 
 ###### 3.2.4.2.3. Errors
 
-- 🔴 Accessing this route without the `Authorization` header returns a [missing_authorization_header](0061-error-format-and-definitions.md#missing_authorization_header) error.
-- 🔴 Accessing this route without the master key returns an [invalid_api_key](0061-error-format-and-definitions.md#invalid_api_key) error.
+- 🔴 Accessing this route without the `Authorization` header returns a [missing_master_key](0061-error-format-and-definitions.md#missing_master_key) error.
+- 🔴 Accessing this route without the right master key returns an [invalid_api_key](0061-error-format-and-definitions.md#invalid_api_key) error.
```

Comment:
We should keep the previous message and just add a previous line mentioning something like

"Accessing this route while a master key is not set for the instance returns a [missing_master_key](0061-error-format-and-definitions.md#missing_master_key) error".